database/seeds/07_flagged_reason.js

```diff
@@ -0,0 +1,16 @@
+
+exports.seed = function(knex) {
+  // Deletes ALL existing entries
+  return knex('flagged_reason').del()
+    .then(function () {
+      // Inserts seed entries
+      return knex('flagged_reason').insert([
+        {reason: 'Spam', weight: 1},
+        {reason: 'Bullying or Harrasment', weight: 2},
+        {reason: 'Hate Speech or Symbols', weight: 3},
+        {reason: 'Nudity or Sexual Content', weight: 4},
+        {reason: 'I just dislike it', weight: 5},
```

Comment:
Not sure that "I just dislike it" deserves to be weighted heavier than hate speech or sexual content. That being said, don't know how we should weigh other...

Reply:
These weights are only placeholders to write the functionality but the weight thing has been on my mind... Question is, how are we going to decide what offense is considered more serious than another? This could be considered a sensitive subject. I certainly do not want to make the decision solo lol I think we should decide as a group. 

That being said, adjusting it to a baseline with 'other' and 'I just dislike it' being 1 all others being 2, so we can write the functionality and set the weights later when we come to a decision.